Duckling/Time/EN/Rules.hs

```diff
@@ -123,6 +123,24 @@ ruleYYYYNamedMonthDOM = Rule
       _ -> Nothing
   }
 
+ruleNamedMonthDOMYYYY :: Rule
+ruleNamedMonthDOMYYYY = Rule
+  { name = "<named-month> <day-of-month> <year>"
+  , pattern =
+    [ Predicate isAMonth
+    , regex "\\s*?\\-\\s*?|\\s*?/\\s*?|\\s*?_+\\s*?|\\s*?and\\s*?|\\s*?to\\s*?|\\s*?th?ru\\s*?|\\s*?through\\s*?|\\s*?(un)?til(l)?\\s*?|\\s+"
+    , Predicate isDOMValue
+    , regex "\\s*?\\-\\s*?|\\s*?/\\s*?|\\s*?_+\\s*?|\\s*?and\\s*?|\\s*?to\\s*?|\\s*?th?ru\\s*?|\\s*?through\\s*?|\\s*?(un)?til(l)?\\s*?|\\s+"
```

Comment:
I think you can make this more readable by extracting the common `\\s*`, it should be enough to have it at the beginning and the end

Reply:
Cool, seems like fixed that.